src/drivers/dw/dma.c

```diff
@@ -702,7 +702,7 @@ static int dw_dma_set_config(struct dma *dma, int channel,
 
 	/* default channel config */
 	chan->direction = config->direction;
-	chan->timer_delay = config->timer_delay;
+	chan->timer = config->timer;
```

Comment:
Now this confuses me, aren't you assigning a boolean value to a time-interval variable? You really mean a delay of 1?